src/jobs/submanPipelineJob.groovy

```diff
@@ -1,39 +1,33 @@
 import jobLib.rhsmLib
-String baseFolder = rhsmLib.submanJobFolder
 
-pipeline_helper = job("$baseFolder/subscription-manager-pipeline-helper") {
-    logRotator {
-        numToKeep(1)
-    }
-    steps {
-        downstreamParameterized {
-            trigger('subscription-manager') {
-                parameters {
-                    predefinedProp('sha1', '${sha1}')
-                    predefinedProp('ghprbActualCommit', '${ghprbActualCommit}')
-                    predefinedProp('ghprbPullId', '${ghprbPullId}')
-                }
-            }
-        }
-    }
-}
+String baseFolder = rhsmLib.submanJobFolder
 
-pipeline = pipelineJob("$baseFolder/subscription-manager") {
-    description('Delivery Pipeline for subscription-manager')
-    parameters {
-        stringParam('sha1', 'master', 'GIT commit hash of what you want to test.')
-        stringParam('ghprbActualCommit', null, 'commit used to report status against a GitHub PR')
-        stringParam('ghprbPullId', null, 'PR number (used in certain checks)')
+multibranchPipelineJob("${baseFolder}/subscription-manager") {
+  branchSources {
+    // upstream; PRs only
+    github {
+      id('35f9f740-fc4b-11ea-a644-482ae3184260') // IMPORTANT: use a constant and unique identifier
+      scanCredentialsId('github-api-token-as-username-password')
+      repoOwner('candlepin')
+      repository('subscription-manager')
+      buildForkPRHead(false)
+      buildForkPRMerge(true)
+      buildOriginBranch(false)
+      buildOriginBranchWithPR(false)
+      buildOriginPRHead(false)
+      buildOriginPRMerge(true)
     }
-    logRotator {
-        numToKeep(20)
+    // downstream; release branches
+    git {
+      remote('https://gitlab.sat.engineering.redhat.com/satellite6/subscription-manager.git')
+      id('b509c5c8-fce7-11ea-a16b-0050b6b571b4')
+      includes('RHEL* SLE_*')
```

Comment:
should we also include the master branch here?

Reply:
I think since we'll have our other test suite running and we'll have these tests run on all PRs it's ok not to test master.